share/github-backup-utils/ghe-backup-mssql

```diff
@@ -180,6 +180,9 @@ if [ -n "$backup_command" ]; then
   ghe-ssh "$GHE_HOSTNAME" -- "$backup_command" || failures="$failures mssql"
   bm_end "$(basename $0)"
 
+  # Configure the backup cadence on the appliance, which is used for diagnostics
+  ghe-ssh "$GHE_HOSTNAME" "ghe-config mssql.backup.cadence $GHE_MSSQL_BACKUP_CADENCE"
```

Comment:
Have to change the script to handle this format.

Reply:
So we consider making our live easier by change this to record the info we need.  The hours/days that back should happen.

Reply:
I'd rather have it be the same everywhere and parse it when we use it.

Reply:
Works for me.